package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remix-auth",
-  "version": "3.5.1",
+  "version": "3.6.0",
```

Comment:
This is changed automatically with a GitHub Action
```suggestion
  "version": "3.5.1",
```

Reply:
Done!  Thanks for the feedback